### PR TITLE
Calendar added

### PR DESCRIPTION
**Pull request by Anuj-krSingh**
sdp-calendar

### CODE REVIEW

File: src/components/templates/TemplateSpotDetail.vue [hunk 1]
```diff
@@ -96,7 +96,11 @@
                             <tr v-if="spotDetails.UpdatedAt">
                                 <td>Updated At</td>
                                 <td>
-                                    {{ spotDetails.UpdatedAt.split('T')[0] }}
+                                    <AtomDatePicker
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSpotDetail.vue:99]: It should show the updated at value by default
And should only show date picker when clicked on it.

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/atoms/AtomDatePicker.vue [hunk 2]
```diff
@@ -25,13 +27,22 @@ export default {
...
+                this.assignedDate != ''
+                    ? new Date(this.assignedDate)
+                    : new Date(),
         };
     },
     methods: {
         onChange(val) {
             this.$emit('changed', val);
         },
+        istFormatter(date) {
```
Comment (NextIshwar) [inline src/components/atoms/AtomDatePicker.vue:40]: This variable name suggests a boolean return value, but it actually returns a date. Could we rename it to something more meaningful?

Comment (arrogant-ant) [reply src/components/atoms/AtomDatePicker.vue:40]: its 'ist'Formatter  (Indian Standard Time)

**Review by NextIshwar — APPROVED** [review]
LGTM

**Pull request merged**